### PR TITLE
Add useful helm chart scripts

### PR DESCRIPTION
**Pull request by jesusvazquez**
This commit is adding two new scripts to:
- Bootstrap a new kind cluster with mimir-distributed installed.
- Generate an enterprise tenant token given a mimir-distributed
  installation with enterprise enabled.

### CODE REVIEW

File: operations/helm/charts/mimir-distributed/scripts/bootstrap_kind_cluster.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+echo "# Creating new kind cluster"
+kind create cluster --name mimir-demo
+
+# OSS Mimir
+helm upgrade --install mimir-demo \
+  "${chart_path}" \
+  --values "${chart_path}/values.yaml" \
+  --kube-context kind-mimir-demo
+
+# Enterprise and graphite enabled
```
Comment (krajorama) [inline operations/helm/charts/mimir-distributed/scripts/bootstrap_kind_cluster.sh:27]: Super weird to have this commented out. I'd suggest to make the license file a CLI flag, then the script can decide what to do.

**Review by krajorama — CHANGES_REQUESTED** [review]
Cool. Is this for internal use or intended for anybody?

File: operations/helm/charts/mimir-distributed/scripts/test_graphite_write_path.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+"tags": ["sponsor=Deutsche_Telekom", "server=Hamburg"]
+}]'
+
+data=${data//REPLACE_DATE/${timestamp}}
+
+curl \
+  -X POST \
+  -H "Authorization: Basic ${MIMIR_BASIC_AUTH}" \
+  -H "Content-Type: application/json" \
+  "${MIMIR_GATEWAY_HOST}:8080/graphite/metrics" -d  "${data}"
```
Comment (dimitarvdimitrov) [inline operations/helm/charts/mimir-distributed/scripts/test_graphite_write_path.sh:50]: This should change the exit code of curl when the status code is >= 400
```suggestion
  --fail-with-body \
  "${MIMIR_GATEWAY_HOST}:8080/graphite/metrics" -d  "${data}"
```

File: operations/helm/charts/mimir-distributed/scripts/test_graphite_read_path.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  echo "Environment variable MIMIR_BASIC_AUTH is not set. Exiting..."
+  exit 1
+fi
+
+echo "# Getting tags"
+curl \
+  -X GET \
+  -H "Authorization: Basic ${MIMIR_BASIC_AUTH}" \
+  -H "Content-Type: application/json" \
+  "${MIMIR_GATEWAY_HOST}:8080/graphite/tags"
```
Comment (dimitarvdimitrov) [inline operations/helm/charts/mimir-distributed/scripts/test_graphite_read_path.sh:26]: This should change the exit code of curl when the status code is >= 400
```suggestion
  --fail-with-body \
  "${MIMIR_GATEWAY_HOST}:8080/graphite/tags"
```

**Review by dimitarvdimitrov — APPROVED** [review]
Hey, sorry I dropped the ball on this one. LGTM with two nitpicks to make these scripts easier to use in CI

@krajorama these are internal for now. I am planning on making them run as a test hook when users do `helm test` and graphite is enabled.

**Comment by jesusvazquez** [comment]
@dimitarvdimitrov addressed your feedback and @krajorama 's as well

**Review by dimitarvdimitrov — APPROVED** [review]

**Review by krajorama — APPROVED** [review]

**Pull request merged**